common/ideologies/00_ig_ideologies_flavored.txt

```diff
@@ -0,0 +1,507 @@
+﻿ideology_venician_patrician = {
```

Comment:
Replacing vanilla files should be avoided where possible. In #67 I've added a separate file for dvg ideologies, so once that merges I'd recommend moving your added ideologies into that file instead of replacing the vanilla file. See here for what I mean: https://github.com/4hrue2kd83f/divergences/blob/80fdcd118c6c4c07cc943d6559b9da875cbe9f66/common/ideologies/00_dvg_ig_ideologies.txt

Reply:
I'll write it down to change it once the Incan commits come in